app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Giftmessage.php

```diff
@@ -74,4 +74,14 @@ protected function _getGiftmessageSaveModel()
     {
         return Mage::getSingleton('adminhtml/giftmessage_save');
     }
+
+    public function canDisplayGiftmessage()
+    {
+        if (!Mage::helper('core')->isModuleEnabled('Mage_GiftMessage')) {
+            return false;
+        }
+        /** @var Mage_GiftMessage_Helper_Message $helper */
+        $helper = $this->helper('giftmessage/message');
+        return $helper->getIsMessagesAvailable('order', $this->getEntity(), $this->getEntity()->getStoreId());
+    }
```

Comment:
If the check for module is in the helper:
```suggestion
    public function canDisplayGiftmessage()
    {
        /** @var Mage_GiftMessage_Helper_Message $helper */
        $helper = $this->helper('giftmessage/message');
        return $helper
            ? $helper->getIsMessagesAvailable('order', $this->getEntity(), $this->getEntity()->getStoreId())
            : false;
    }
```